recipe/meta.yaml

```diff
@@ -33,11 +33,13 @@ outputs:
         - python >=3.6
         - pkg-config
         - {{ compiler('c') }}
+        - {{ stdlib('c') }}
         - glib  # [build_platform != target_platform]
         # Required for the conda-forge meson cross-compilation config to work
         - {{ compiler('cxx') }}
       host:
         - glib
+        - libintl-devel  # [win]
```

Comment:
Normally, this should also be needed on `osx`.

Reply:
There's a DSO warning in the last build, so it makes sense to add it now as a dep.